src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentOneDay = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  const totalPrice = days * rentOneDay;
+
+  if (days >= 3 && days < 7) {
+    return totalPrice - discountThreeDays;
+  } else if (days >= 7) {
+    return totalPrice - discountSevenDays;
+  }
+
+  return totalPrice;
```

Comment:
it is not best practice to use `else if` statements here, let's use each check separately